### PR TITLE
Change settings handling

### PR DESCRIPTION
**Pull request by alchris**
The syntax of the SAML_AUTH settings dict is now identical to the underlying python3-saml plugin (https://github.com/onelogin/python3-saml).
This offers more configuration options.  Furthermore the plugin is so automatically compatible to future versions of python3-saml

### CODE REVIEW

File: back/taiga_contrib_saml_auth/settings.py [hunk 1]
```diff
@@ -1,41 +1,42 @@
...
-                },
-            'security': settings.SAML_AUTH.get('security', {}),
-            'organization': settings.SAML_AUTH.get('organization', {}),
-            'contactPerson': settings.SAML_AUTH.get('contactPerson', {}),
-            }
+        'strict': not debug,
+        'debug': debug,
+        'sp': {
+            'entityId': base_url + reverse('taiga_contrib_saml_auth:metadata'),
+            'assertionConsumerzService': {
```
Comment (jgiannuzzi) [inline back/taiga_contrib_saml_auth/settings.py]: This should remain `'assertionConsumerService'`

File: README.md [hunk 2]
```diff
@@ -184,106 +208,8 @@ workon taiga
 pip install -e .
 ```
 
-Modify `taiga-back/settings/local.py` and include the line:
+Modify `taiga-back/settings/local.py`. See [Taiga Back](#Taiga Back) section for content:
```
Comment (jgiannuzzi) [inline README.md]: Excellent idea, but unfortunately that link does not seem to work.
I would suggest writing `See "Production env -> Taiga Back" section for content.`

**Comment by jgiannuzzi** [comment]
Hi @alchris, thanks a lot for your PR.

Could you please rewrite your two commits as per my comments and then force push them to your branch?

**Comment by alchris** [comment]
I made the changes as requested

**Comment by jgiannuzzi** [comment]
Merged, thanks @alchris!

**Pull request merged**